src/consent-manager-builder/index.tsx

```diff
@@ -176,42 +183,53 @@ export default class ConsentManagerBuilder extends Component<Props, State> {
     }
   }
 
-  initialise = async () => {
+  initialise = async (nextProps = undefined) => {
     const {
       writeKey,
       otherWriteKeys = ConsentManagerBuilder.defaultProps.otherWriteKeys,
+      otherDestinations = ConsentManagerBuilder.defaultProps.otherDestinations,
       shouldRequireConsent = ConsentManagerBuilder.defaultProps.shouldRequireConsent,
       initialPreferences,
       mapCustomPreferences,
       defaultDestinationBehavior,
       cookieName,
       cdnHost = ConsentManagerBuilder.defaultProps.cdnHost
-    } = this.props
+    } = nextProps || this.props
```

Comment:
Do we need these `nextProp` changes, since you took out the `componentWillReceiveProps`?

Reply:
we most definitely do not, good catch!